src/style.css

```
@@ -0,0 +1,113 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  padding: 0 14px;
+  width: 200px;
+  height: 403px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  text-align: center;
+}
+
+.card__picture {
+  height: 134px;
+  width: 160px;
+  margin: 29px 0 41px;
+}
+
+.card__title {
+  text-align: left;
+}
+
+.card__name {
+  margin: 0 0 6px;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 15px;
+}
+
+.card__code {
+  margin: 0 0 16px;
+  font-size: 11px;
+  font-weight: 400;
+  line-height: 13px;
+}
+
+.card__raiting {
+  margin: 0 0 25px;
+  font-size: 11px;
+  font-weight: 400;
+  line-height: 13px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  margin-right: 4px;
+  width: 15px;
+
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: 50%;
+}
+
+.stars--4 .stars__star:nth-child(-n + 4) {
+  background-image: url(images/star-active.svg);
+}
+
+.card__price {
+  margin: 0 0 16px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.card__price_tag {
+  font-size: 11px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 19px;
+  vertical-align: bottom;
+}
+
+.card__price_size {
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 13px;
+  text-align: right;
+}
+
+.card__link {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+
+  height: 40px;
+  border-radius: 5px;
+  line-height: 16px;
+  font-size: 14px;
+  font-weight: 700;
+  font-style: normal;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #fff;
+  background: #00acdc;
+}
+
+.card__link:hover {
+  color: #00acdc;
+  background-color: #fff;
+  border: 1px solid #00acdc;
```

```suggestion
  border-color: #00acdc;
```
no need to change the whole border, if you need to change only border-color

---

The problem was that in the previous block link I did not have a border, but now i added it. Thanks